components/ConvSnippet.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from 'react';
+import { allAuthors, Authors as Author } from 'contentlayer/generated';
+import Image from 'next/image';
+
+export interface ConvSnippetProps {
+  name: string;
+  children: ReactNode;
+}
+
+const findAuthor = (name: string): Author => {
+  for (const author of allAuthors) {
+    if (author.name == name) {
+      return author;
+    }
+  }
+
+  return allAuthors[0];
+};
+
+export const ConvSnippet = ({ name, children }: ConvSnippetProps) => {
+  const author = findAuthor(name);
+  const handle = author.name.split(' ', 1)[0];
+
+  return (
+    <div className="flex space-x-4 rounded-lg bg-gray-100 p-4">
+      <div className="flex h-16 w-16 shrink-0 items-center justify-center self-center overflow-hidden rounded-lg bg-gray-200">
```

Comment:
Addresses dark theme: 
![image](https://user-images.githubusercontent.com/40265/214885842-43bd881b-f7e9-470e-9827-277e100fa017.png)


```suggestion
    <div className="flex space-x-4 dark:bg-gray-800 rounded-lg bg-gray-100 p-4">
      <div className="flex h-16 w-16 shrink-0 items-center justify-center self-center overflow-hidden rounded-lg dark:bg-gray-700 bg-gray-200">
```